app/models/deidentified_covid_pathways_assessment.rb

```diff
@@ -14,4 +14,51 @@ def title
   def default?
     false
   end
+
+
+  def assessment_params(params)
+    params.require(:identified_covid_pathways_assessment).permit(
+      [
+        :entry_date,
+        :phone_number,
+        :email_addresses,
+        :shelter_name,
+        :case_manager_contact_info,
+        :mailing_address,
+        :day_locations,
+        :night_locations,
+        :other_contact,
+        :household_size,
+        :pending_subsidized_housing_placement,
+        :pending_housing_placement_type_other,
+        :income_total_annual,
```

Comment:
Let's pull out any of these that are shared into a method on non_hmis_assessment so we don't forget to add them in multiple places in the future.